### ISSUE TITLE
Building in a folder named "fw" doesn't work

### ISSUE THREAD

**Issue by icarroll-intrepid** [cesanta/mongoose-os#331]
Building in a folder named "fw" will cause an error.
```
ubuntu@ip-172-31-5-64:~$ mkdir fw
ubuntu@ip-172-31-5-64:~$ cd fw
ubuntu@ip-172-31-5-64:~/fw$ mos init
Downloading empty app...
Unpacking...
ubuntu@ip-172-31-5-64:~/fw$ sudo mos build --arch cc3200
[...]
Error: open build/objs/fw.elf: no such file or directory
Error: build failed
```

**Comment by icarroll-intrepid** [comment]
Sorry, I hit enter before I finished typing the title.

**Comment by dimonomid** [comment]
Thanks for reporting, fixed in latest, will be included in the upcoming release 1.15.
